apps/dolly-backend/src/main/java/no/nav/dolly/bestilling/brregstub/BrregstubClient.java

```diff
@@ -9,27 +9,25 @@
 import no.nav.dolly.domain.jpa.BestillingProgress;
 import no.nav.dolly.domain.resultset.RsDollyUtvidetBestilling;
 import no.nav.dolly.domain.resultset.tpsf.DollyPerson;
-import no.nav.dolly.errorhandling.ErrorStatusDecoder;
 import no.nav.dolly.service.DollyPersonCache;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Service;
 
 import java.util.List;
-import java.util.stream.Collectors;
 
 import static java.util.Objects.nonNull;
+import static no.nav.dolly.errorhandling.ErrorStatusDecoder.encodeStatus;
+import static org.apache.commons.lang3.StringUtils.isBlank;
 
 @Slf4j
 @Service
 @RequiredArgsConstructor
 public class BrregstubClient implements ClientRegister {
 
     private static final String OK_STATUS = "OK";
+    private static final String NOK_STATUS = "Feil= ";
```

Comment:
Er det tenkt Not Ok Status? mulig jeg bare er litt tett altså men synes kanskje ERROR_STATUS eller lignende er lettere å tyde som at noe har gått galt 😅

Reply:
Enig, endrer dette.